kore/src/main/scala/org/kframework/definition/outer.scala

```diff
@@ -593,6 +607,8 @@ object Production {
   }
 
   val kLabelAttribute = "klabel"
+  private var uid = 1
```

Comment:
I don't like the insertion of statefulness here. It means that the grammar you generate is going to be different each time you run something, depending on what has been run previously. Can we please modify this so that the grammar generated is deterministically determined by the input module?